packages/backend/src/libp2p/connectionsManager.ts

```diff
@@ -255,12 +254,15 @@ export class ConnectionsManager extends EventEmitter {
         community.peers = sortedPeers
       }
       await this.localStorage.put(LocalDBKeys.COMMUNITY, community)
+      if ([ServiceState.LAUNCHING, ServiceState.LAUNCHED].includes(this.communityState)) return
+      this.communityState = ServiceState.LAUNCHING
       await this.launchCommunity(community)
```

Comment:
This should be moved to the bottom, first we should set status to launching for both community and registrar, then launch them